game1class.py

```diff
@@ -0,0 +1,25 @@
+import random
```

Comment:
*Siempre* debéis trabajar con PRs que revise alguien (o varios) del resto del equipo.